app/javascript/controllers/navbar_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="navbar"
+export default class extends Controller {
+  connect() {
+    document.getElementById("drop").addEventListener("click", function(e) {
+      document.getElementById("myDropdown").classList.toggle("show");
+    });
```

Comment:
We don't use getElementById or document.querySelector or addEventListener with Stimulus.
Use data-action and data-controller-target instead in the html please
Here's an example
![image](https://user-images.githubusercontent.com/75388869/219660351-10039c9f-7544-4de7-b2a2-bfa79706d3f6.png)